api/test/wfl/integration/executor_test.clj

```diff
@@ -104,80 +104,114 @@
   {:referenceId snapshot-reference-id
    :name        snapshot-reference-name})
 
-(def ^:private method-config-version-mock 1)
+;; Method configuration
```

Comment:
And thus begins mock salad…

It's no fun to work with all of these helpers and I've only added to the complexity.  We're trying to mock out external service calls, of which there are many.  I do welcome feedback on how to streamline to make this space easier to work in in the future.

Reply:
I faced the same problem with testing the Slack notification, and really mocking out those function adds a huge amount of complexity and sometimes doesn't really add too much value in terms of testing. I had to refactor a lot of functions so mock is minimally required. But in this case of talking to Firecloud/Rawls, I cannot think of a better way yet.